docs/technical-docs/customisation.md

```diff
@@ -0,0 +1,245 @@
+---
+sidebar_position: 2
+title: Customisation
+---
+
+# Customisation Guide
```

Comment:
@gerceboss Somewhere in the technical docs (probably at the start), highlight the fact that these docs are still in active development and might change over time.